### PR TITLE
dashboard c+view missing style

### PR DESCRIPTION
**Pull request by mynlexi**
basic dashboard controller for an overview
no buttons

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+<div class="container", style="display: flex">
+  <div class="container">
+    <h2>Your Tanks</h2>
+    <% @tanks.each do |tank| %>
+      <div class="tank-card">
+        <%= tank.name  %>
+        <%= tank.description  %>
+        <br>
+        <%= tank.price_per_day %> | <%= tank.price_per_day %>
```
Comment (Tim0Cc) [inline app/views/dashboard/index.html.erb:14]: warum zweimal price per day?

Comment (mynlexi) [reply app/views/dashboard/index.html.erb:14]: lul

**Comment by mynlexi** [comment]
@VidalRoman now its fucked because u merged without looking

**Comment by VidalRoman** [comment]
I did validate every single line!!!

**Pull request merged**